projects/plugins/jetpack/tools/webpack.config.css.js

```diff
@@ -123,7 +123,7 @@ const weirdRtlEntries = {
 		// When making changes to that list, you must also update $concatenated_style_handles in class.jetpack.php.
 		'modules/carousel/swiper-bundle.css',
 		'modules/carousel/jetpack-carousel.css',
-		'modules/contact-form/css/grunion.css',
+		'jetpack_vendor/automattic/jetpack-forms/src/contact-form/css/grunion.css',
```

Comment:
@monsieur-z should this be updated as well? https://github.com/Automattic/jetpack/pull/35091/files#diff-6da16705e102442c40be71800880c9f59273c2ed3b17a01b5f59efa8182761acR192

Reply:
I don't think so, since that's only used along with the other code in `projects/plugins/jetpack/modules/contact-form/`. But I think a good follow-up to this PR would be to start getting rid of all that in favor of the forms package.